apps/election-manager/src/config/types.ts

```diff
@@ -88,17 +88,17 @@ export interface ContestTallyMeta {
 export type ContestTallyMetaDictionary = Dictionary<ContestTallyMeta>
 
 export interface Tally {
-  readonly precinctId?: string
-  readonly scannerId?: string
   readonly numberOfBallotsCounted: number
-  readonly castVoteRecords: ReadonlyMap<string, CastVoteRecord>
+  // TODO(#2975): Once we're removing duplicate ballots, make this a dictionary indexed by ballotId
```

Comment:
I had this as a Map from ballotId to CVR but it wasn't really used until this change. Knowing that we currently have a bug with ballotIds I don't want to introduce not being able to handle duplicate ballotIds (when today we just count both votes) right before the freeze. Temporarily making this a list (which makes the filtering code a bit less performant) to keep things closer to existing behavior for now but I will eventually change this back and add in proper error/warning messages for duplicate ballot ids, etc.